app/src/main/java/nodomain/freeyourgadget/gadgetbridge/devices/miband/MiBand2Coordinator.java

```diff
@@ -47,6 +47,8 @@
 import nodomain.freeyourgadget.gadgetbridge.model.DeviceType;
 import nodomain.freeyourgadget.gadgetbridge.util.Prefs;
 
+import static nodomain.freeyourgadget.gadgetbridge.GBApplication.getContext;
```

Comment:
Thats not good

Reply:
Was just following the HPlusCoordinator, from where I based my code. Probably should have stuck to the MiBand2Coordinator style though.

Reply:
please just call GBApplication.getContext() and remove the import :)